src/json_parsing/test_fixtures/crashes.json

```diff
@@ -2,7 +2,7 @@
   "nextLink": "/api/v0.1/apps/XING-SE-Organization/XING/errors/errorGroups?version=7.41.3&start=2019-05-15&end=2019-05-20&$orderby=count%20desc&$top=30&$token=123456789",
   "errorGroups": [
     {
-      "errorGroupId": "92620314u",
+      "errorGroupId": "92620314",
```

Comment:
Just wondering if, for future generations, it would be less confusing to rename this file to `api_response.json` since it's a mixture of crashes and errors.